### PR TITLE
Dataviews get bounding box from map object

### LINKED ISSUE #1158: [vector] Result data (using widgets) in vector rendering differs with raster data
### STR

- Open populated-places example in DI (raster way).
- Open populated-places example in DI with vector enabled (in another window).
- Compare results from widgets, they are totally different in all widgets.

untitled

Assign it accordingly @rochoa @fdansv @alonsogarciapablo

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Partially fixes #1158. The bounding box that widgets were using to get information from Windshaft was not right.

@xavijam please review. Thanks!

cc: @fdansv

### CODE REVIEW

File: src/dataviews/dataview-model-base.js [hunk 1]
```diff
@@ -127,6 +130,10 @@ module.exports = Model.extend({
         silent: silent,
         forceFetch: forceFetch
       });
+
+      if (this.get('sync_on_data_change')) {
+        this._newDataAvailable = true;
```
Comment (xavijam) [inline src/dataviews/dataview-model-base.js]: What is this?

Comment (alonsogarciapablo) [inline src/dataviews/dataview-model-base.js]: That flag is used [here](https://github.com/CartoDB/cartodb.js/pull/1162/files#diff-805364e485d740e26e33839755aab753R161).

**Comment by alonsogarciapablo** [comment]
Ping!

**Comment by xavijam** [comment]
:dancer: :+1: :shipit:

**Pull request merged**